UPGRADE-5.0.md

```diff
@@ -26,6 +26,7 @@ The underlying library used for sending emails (Swift Mailer) was discontinued a
       * `.env.$APP_ENV`       committed environment-specific defaults
       * `.env.$APP_ENV.local` uncommitted environment-specific overrides
     * The system run similar index_dev.php if you use `APP_ENV=dev` and `APP_DEBUG=1` in your .env.local file.
+    * Remove Custom shortener API URL from configuration (link_shortener_url variable) and remove `Mautic\CoreBundle\Helper\UrlHelper::getShortenerUrl()` method. Use the `\Mautic\CoreBundle\Shortener\Shortener::class` service instead.
```

Comment:
Please describe in the "User facing changes" what users need to change to migrate from the old shortener configuration to the new one.

Reply:
I update it. 
I am gonna write user doc

Reply:
User docs https://github.com/mautic/mautic/pull/12299

Reply:
https://github.com/mautic/mautic/pull/12299

Reply:
The link goes to this PR.

Reply:
https://github.com/mautic/user-documentation/pull/191

Reply:
Thanks! Could you please add a note to the "User Facing changes" so users who were using the shortener would know they have to reconfigure it when coming from M4 to M5?

Reply:
I add notice to doc 

https://github.com/mautic/user-documentation/pull/191/files

Reply:
We need a list of things that the users must check before they upgrade from M4 to M5. Could you please add a note that they'll have to reconfigure the shortener?